frontend/src/components/organisms/Header/index.jsx

```diff
@@ -0,0 +1,55 @@
+import "./index.less";
+
+import {
+  Brand,
+  Masthead,
+  MastheadBrand,
+  MastheadContent,
+  MastheadMain,
+  MastheadToggle,
+  PageToggleButton,
+} from "@patternfly/react-core";
+import { useDispatch, useSelector } from "react-redux";
+
+import { BarsIcon } from "@patternfly/react-icons";
+import logo from "@/assets//logo.png";
+import { showToast } from "@/actions/toastActions";
+import { toggleSideMenu } from "@/actions/sideMenuActions";
+
+const Header = () => {
+  const isSideMenuOpen = useSelector((state) => state.sidemenu.isSidebarOpen);
+
+  const dispatch = useDispatch();
+  const onSidebarToggle = () => {
+    dispatch(toggleSideMenu(false));
+    dispatch(showToast("success", "Hello"));
```

Comment:
nit: Please remove this message.

Reply:
Addressed in PR[102](https://github.com/cloud-bulldozer/cpt-dashboard/pull/102)